src/components/util/hexToRgb.ts

```diff
@@ -0,0 +1,12 @@
+
+
+export default function hexToRgb(hex: string, opacity: number): string {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+
+  const BASE_16 = 16;
```

Comment:
let's define the const outside the function. otherwise, it gets created every time the function runs. its an improvement